forage-android/src/main/java/com/joinforage/forage/android/collect/VGSPinCollector.kt

```diff
@@ -89,10 +90,14 @@ internal class VGSPinCollector(
                     )
                     is VGSResponse.ErrorResponse -> continuation.resumeWith(
                         Result.success(
-                            ForageApiResponse.Failure(response.body.orEmpty())
+                            ForageApiResponse.Failure(response.errorCode, listOf(ForageError(response.errorCode, "user_error", "Invalid Data")))
+                        )
+                    )
+                    null -> continuation.resumeWith(
+                        Result.success(
+                            ForageApiResponse.Failure(500, listOf(ForageError(500, "server_error", "Unknown Server Error")))
```

Comment:
`unknown_server_error` would be a valid code from our backend

Reply:
All error codes have been updated here: [7b8dd2b](https://github.com/teamforage/forage-android-sdk/pull/23/commits/7b8dd2baa3cb17edf7cf883683e0fc21e3f2c654)